src/metrics/meter.rs

```diff
@@ -20,6 +21,13 @@ pub struct MeterSnapshot {
     pub mean: f64,
 }
 
+impl Display for MeterSnapshot {
+    fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
+        write!(f, "{}, {}", self.count, self.mean)
```

Comment:
Its so you can do stuff like `println!("{}", metric)` as opposed to `println!("{:?}", metric)`, different things can still display however they want.